array/DNAm/preprocessing/normalisation.r

```diff
@@ -44,6 +44,8 @@ setwd(dataDir)
 
 gfile<-openfn.gds(gdsFile, readonly = FALSE)
 
+probeFilt <- with(new.env(), {load(file.path(qcOutFolder, "QCmetrics.rdata")); get("probeFilt")})

```

Comment:
I can see why you are doing this. I couldn't find a nicer solution than this line so we'll keep it.